src/main/java/tech/thatgravyboat/repolib/api/AttributesAPI.java

```diff
@@ -0,0 +1,37 @@
+package tech.thatgravyboat.repolib.api;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public final class AttributesAPI {
+
+    private final Map<String, JsonObject> attributes = new HashMap<>();
```

Comment:
these should have an internal representation like runes and pets that gets made from the json.